CHANGELOG.md

```diff
@@ -2,10 +2,31 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.6 August 11, 2021
+# 10.0.0-rc.7 August 25, 2021
 
 **The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
 
+## Features ✨ and improvements 🏁
+* Add generateId property for GeoJsonSource. ([#538](https://github.com/mapbox/mapbox-maps-android/pull/538))
+* Add default value to improve usability of FeatureState API. ([#588](https://github.com/mapbox/mapbox-maps-android/pull/588))
+* Added move layer API
+
+## Bug fixes 🐞
+* Fix snapshotter leak. Now calling MapSnapshotter#destroy truly frees memory. ([#569](https://github.com/mapbox/mapbox-maps-android/pull/569))
+* Remove the expression getter/setters for source properties. ([#568](https://github.com/mapbox/mapbox-maps-android/pull/568))
```

Comment:
@pengdev please remind me if we're considering it as a breaking change.

Reply:
Yes, I think it's a breaking change